infrastructure/server/waf-firewall.tf

```diff
@@ -0,0 +1,226 @@
+locals {
+    # Example: "fam-dev-waf"
+    web_acl_name_prefix = "fam-${var.target_env}-waf"
+}
+
+
+# API-Gateway WAF ACL
+locals {
```

Comment:
Not sure why there are two locals blocks, can't this be combined into one.

Reply:
Yes, they can be combined.
I chose it to be separated since Terraform has feature for grouping "locals" for the same category and references "**common locals**" in different "locals" blocks so I define this locals to be close to the API-Gateway rule resource block where the value is actually being referenced.

Reply:
Ah, okay so this is really the WAF resources for the backend, while the other is the WAF resources for the front-end.